src/modules/rover_ackermann/RoverAckermannGuidance/RoverAckermannGuidance.cpp

```diff
@@ -201,6 +202,13 @@ void RoverAckermannGuidance::updateWaypointsAndAcceptanceRadius()
 	} else {
 		_acceptance_radius = _param_nav_acc_rad.get();
 	}
+
+	if (position_setpoint_triplet.current.cruising_speed > 0.f) {
+		_wp_max_desired_vel = math::min(position_setpoint_triplet.current.cruising_speed, _param_ra_miss_vel_def.get());
```

Comment:
```suggestion
		_wp_max_desired_vel = position_setpoint_triplet.current.cruising_speed;
```
There is no reason not to allow the desired cruising speed to be higher than the default mission speed.

Reply:
It's a security in case the user make a mistake when creating the mission (for exemple writing 32m/s instead of 3.2m/s) so the rover don't go at a crazy speed and become dangerous. 

And sometimes a higher speed can introduce oscillations if the rover have a slew rate too low.

And if the user know what he do and he really want a higher speed he can change the parameter.

Maybe we should use RA_MAX_SPEED instead ?
But from what I understood it is used to determine the throttle. And if we want to limit the speed under the max possible speed of the rover, then it wouldn't be possible.

So maybe we should create a new param RA_MISS_VEL_MAX to set a max speed in mission, higher than RA_MISS_VEL_DEF and lower than RA_MAX_SPEED ?

Reply:
I don't think we need to make concessions on functionality to account for user errors. The user should have full control over the speed, which includes raising the mission speed from the default to a higher value. There is no reason to stop the user from driving a mission at full throttle.
We can do something like this, but the rover is physically limited to _param_ra_max_speed.get() anyway if it was set properly:
```suggestion
		_wp_max_desired_vel = math::constrain(position_setpoint_triplet.current.cruising_speed, 0.f, _param_ra_max_speed.get());
```